templates/partials/navigation.phtml

```diff
@@ -5,6 +5,13 @@ declare(strict_types=1);
 /**
  * @var League\Plates\Template\Template $this
  */
+$currentUrl = null;
+try {
+    $currentUrl = $this->url();
```

Comment:
Is this the only place where the helper is used in this way?

Reply:
It it the only place the helper was used to get the current URL, yes.

 The error occurred because this partial is included in the default layout (`default.phtml`) which `404.phtml` is extending as well, resulting in the conditionals being run on non-existent pages

Reply:
The problem is clear, I didn't want a place to be overlooked.

Reply:
It is a workaround and I hope that we can find a better solution in the future. Therefore, a new issue report should be created that includes the workaround and then we can merge this pull request.

Reply:
@froschdesign Sorry, by "a new issue report" do you mean creating a new issue altogether, or simply updating/adding a comment to the existing #191 issue describing this workaround?

Reply:
There is a helper that returns actual request url.

```
/**
 * @var League\Plates\Template\Template $this
 */
$currentPath = rtrim(parse_url($this->serverurl(), PHP_URL_PATH), '/');
```